### ISSUE TITLE
error: constructor SQLitePluginPackage in class SQLitePluginPackage cannot be applied to given types

### ISSUE THREAD

**Issue by Moussawi7** [andpor/react-native-sqlite-storage#65]
Hi,
When i try to install the plugin on a fresh **android** project, I got the following error:
```
MainActivity.java:40: error: constructor SQLitePluginPackage in class SQLitePluginPackage cannot be applied to given types;
			new SQLitePluginPackage()
			^
  required: Activity
  found: no arguments
  reason: actual and formal argument lists differ in length
1 error
:app:compileDebugJavaWithJavac FAILED
```

and this is the dependencies:
    "react": "15.2.0",
    "react-native": "0.28.0",
    "react-native-sqlite-storage": "^3.0.0"

Any Idea about the cause of the issue?
Thank you.

**Comment by RakanNimer** [comment]
Hello @andpor and @Moussawi7 !

I had the same issue and it turns out there's a small thing missing in the README.md. I submitted a pull request to clear this up.

In your MainActivity.java you should pass 
``` this ```
to the SQLitePluginPackage : 
```java
    @Override
    protected List<ReactPackage> getPackages() {
        return Arrays.<ReactPackage>asList(
            new MainReactPackage(),
            new SQLitePluginPackage(this)
        );
    }
```

That worked for me, hope it helps and thanks @andpor for the great library !

**Comment by Et3Ali** [comment]
Oh @RakanNimer ,
cool, it worked finally :).

**Comment by Moussawi7** [comment]
Thank you @RakanNimer .
your fix solved my issue.

**Comment by andpor** [comment]
The version of Readme relates to the upcoming release of the code which deprecates the Activity based construction in favor of empty constructor. If you use the latest code from the repo - not released to npm yet (3.1.0) you should be able use an empty constructor. So pull request is not needed..
